mhlib/app.go

```diff
@@ -197,8 +192,10 @@ func (a *App) apply(configFile string, simulate bool) (*[]interface{}, error) {
 	// "performs pods restart for the resource if applicable"
 	cmd = append(cmd, []interface{}{"--recreate-pods"}...)
 
-	// "namespace to install the release into. (Default: 'default')"
-	cmd = append(cmd, []interface{}{"--namespace", a.Namespace}...)
+	// "namespace to install the release into. (Defaults to helm default behaviour => kubeconfig checked out ns)"
+	if a.Namespace == "" {
```

Comment:
@wozniakjan, shouldn't this be `if a.Namespace != "" {` ?

Reply:
absolutely, I even said that in the previous PR... Thanks for spotting this!